src/app/containers/Ad/Canonical/index.jsx

```diff
@@ -1,6 +1,24 @@
 import React from 'react';
+import styled from 'styled-components';
+import { C_LUNAR_LIGHT } from '@bbc/psammead-styles/colours';
+import { GEL_GROUP_4_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
 import AdSlot from './AdSlot';
+import CanonicalAdBootstrap from './CanonicalAdBootstrapJs';
 
-const CanonicalAd = () => <AdSlot uniqueId="leaderboard" />;
+const StyledAdDiv = styled.div`
+  background-color: ${C_LUNAR_LIGHT};
+  min-height: 5.5rem;
```

Comment:
It'd be good to pull out constants to be labelled like this:
```
const LEADERBOARD_HEIGHT = '5.5rem';
```

```suggestion
  min-height: ${LEADERBOARD_HEIGHT};
```

I believe these constant values for should be shared between the Canonical and AMP implementation.

Reply:
Just looking at the AMP implementation now, it seems there we're using pixels there instead of rems: https://github.com/bbc/simorgh/blob/f5243859053990f62aa6072fabf7ffd96b2582fa/src/app/containers/Ad/Amp/index.jsx#L72-L73
I'm not sure why that's the case. I presume these should scale when we zoom in on a page like the rest of the page does. 
So I'd suggest in future work we should have the height and width defined in the parent `src/app/containers/Ad/index.jsx` component and passed down to each of the Canonical and Amp ad components.